client/tests/sdk/test_api.py

```diff
@@ -191,6 +195,131 @@ def test_download_reply(self):
         # Let us remove the temporary directory
         shutil.rmtree(tmpdir)
 
+    @VCRAPI.use_cassette
+    def test_download_submission_autoresume(self):
+        # Get a submision
+        submissions = self.api.get_all_submissions()
+        for s in submissions:
+            if s.is_file():
+                submission = s
+                break
+
+        # Get the submission's content
+        with tempfile.TemporaryDirectory() as tmpdir:
+            _, filepath = self.api.download_submission(submission, tmpdir)
+            with open(filepath, "rb") as fobj:
+                content = fobj.read()
+
+        OriginalPopen = subprocess.Popen
```

Comment:
Can we use [mocking](https://pytest-mock.readthedocs.io/en/latest/index.html) instead of monkey patching?